src/backend/dx12/src/device.rs

```diff
@@ -275,6 +277,7 @@ impl d::Device<B> for Device {
         n::RenderPass {
             attachments: attachments.to_vec(),
             subpasses,
+            dependencies: dependencies.to_vec(),
```

Comment:
Something to keep in mind (for a follow-up) are the implicit dependencies at the start and end of a renderpass to EXTERNAL if there are any.

Reply:
It's actually handled by the PR. Not correctly yet, but handled.